test/testutil.py

```diff
@@ -147,3 +149,270 @@ def create_filesystem_customizations(rootfs: str):
     "-v", "/var/lib/containers/storage:/var/lib/containers/storage",
     "--security-opt", "label=type:unconfined_t",
 ]
+
+
+def get_ip_from_default_route():
+    default_route = subprocess.run([
+        "ip",
+        "route",
+        "list",
+        "default"
+    ], check=True, capture_output=True).stdout
+    return default_route.split()[8].decode("utf-8")
+
+
+@dataclasses.dataclass
+class GPGConf:
+    _key_params_tmpl: str = """
+      %no-protection
+      Key-Type: RSA
+      Key-Length: {key_length}
+      Key-Usage: sign
+      Name-Real: Bootc Image Builder Tests
+      Name-Email: {email}
+      Expire-Date: 0
+    """
+    _base_dir: str = "/tmp/bib-tests"
```

Comment:
Hm, my gut feeling is that we want something like `_base_dir: str = tempfile.mkdtemp(prefix="bib-tests")` here or maybe a small comment why we use a fixed dirname in tmp?

Reply:
`base_dir`  is a simple string and is the default value (chosen from other parts in the code) but notice that the actual directory creation is not handled by the class at any point.
In this case we just use the `output_dir` for each build when `RegistryConf` and `GPGConf` classes are instantiated (see lines 272 and 274 in the `tests_build.py` file). I mean, the actual value of the default `_base_dir` does not really matter as it will not be actually used.
If I use your suggestion an additional temporary directory will be created for each build that will not be actually used.

Reply:
Perhaps it's better than to instead have no default values and error out if any of the getters are used before setting the base?

Reply:
Ok, tried a different/simpler approach, please take a look